ironic-python-agent/0001-Bumping-version-of-debian-to-stretch-adding-user-cor.patch

```diff
@@ -0,0 +1,124 @@
+From 2fe7260df7dc6eecbc82751704840a98c860dd0e Mon Sep 17 00:00:00 2001
+From: PiotrProkop <piotr.prokop@intel.com>
+Date: Thu, 11 Jan 2018 13:34:44 +0100
+Subject: [PATCH] Bumping version of debian to stretch, adding user core to
+ coreos IPA image. Fixing script coreos-oem-inject.py to work on machines with
+ Python3 as default interpreter.
+
+---
+ Dockerfile                             | 20 ++++++++++++++------
+ get_megacli.sh                         | 16 ++++++++++++++++
+ imagebuild/coreos/coreos-oem-inject.py |  3 ++-
+ imagebuild/coreos/oem/cloud-config.yml |  5 +++++
+ 4 files changed, 37 insertions(+), 7 deletions(-)
+ create mode 100755 get_megacli.sh
+
+diff --git a/Dockerfile b/Dockerfile
+index 8c7a56a..d8f02b4 100644
+--- a/Dockerfile
++++ b/Dockerfile
+@@ -1,4 +1,4 @@
+-FROM debian:jessie
++FROM debian:stretch
+ 
+ # The add is before the RUN to ensure we get the latest version of packages
+ # Docker will cache RUN commands, but because the SHA1 of the dir will be
+@@ -13,20 +13,25 @@ ADD . /tmp/ironic-python-agent
+ # 1.6. Using the ARG command will be a much cleaner solution.
+ COPY proxy.sh /usr/bin/proxy.sh
+ 
++ADD get_megacli.sh /tmp/get_megacli.sh
```

Comment:
This should already be available at `/tmp/ironic-python-agent/get_megacli.sh`. That said, I kinda feel like it should be in `imagebuild/common`, rather than the root of the repo.

Reply:
OK